### PR TITLE
i18n(zh-cn): translate `guides/cms/hygraph.mdx`

### PR DESCRIPTION
**Pull request by miroim**
#### What kind of changes does this PR include?

- Translated content

#### Description

translate `\guides\cms\hygraph.mdx`

### CODE REVIEW

**Comment by miroim** [comment]
cc: @liruifengv

File: src/content/docs/zh-cn/guides/cms/hygraph.mdx [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+title: Hygraph & Astro
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/hygraph.mdx:2]:
```suggestion
title: Hygraph 与 Astro
```

File: src/content/docs/zh-cn/guides/cms/hygraph.mdx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+description: 使用 Hygraph 作为 CMS 将內容添加到你的 Astro 项目
+type: cms
+service: Hygraph
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+
+[Hygraph](https://hygraph.com/) 是一个无头 CMS（Content Management System，内容管理系统）。它提供了一个用于 Astro 获取内容的 GraphQL 端点。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/hygraph.mdx:10]:
```suggestion
[Hygraph](https://hygraph.com/) 是一个无头的 CMS（Content Management System，内容管理系统）。它提供了一个用于 Astro 获取内容的 GraphQL 端点。
```

**Review by 100gle — CHANGES_REQUESTED** [review]

**Comment by 100gle** [comment]
LGTM!

**Review by 100gle — APPROVED** [review]

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @miroim 💜

**Pull request merged**